api/app/src/routes/medical/patient.ts

```diff
@@ -50,6 +52,16 @@ router.post(
     const facilityId = getFromQueryOrFail("facilityId", req);
     const payload = patientCreateSchema.parse(req.body);
 
+    if (Config.isSandbox()) {
+      // limit the amount of patients that can be created in sandbox mode
+      const numPatients = await Patient.count({ where: { cxId } });
+      if (numPatients >= Config.SANDBOX_PATIENT_LIMIT) {
```

Comment:
should this be `numPatients > Config.SANDBOX_PATIENT_LIMIT`?

otherwise it will error on 20